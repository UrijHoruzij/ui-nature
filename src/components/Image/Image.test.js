import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import { Image } from "../";
import ThemeProvider from "../../ThemeProvide";

describe("Image component", () => {
  test("Render image", () => {
    const component = (
      <ThemeProvider>
        <Image
          width="300"
          height="400"
          src="https://images.unsplash.com/photo-1477322524744-0eece9e79640"
        />
      </ThemeProvider>
    );
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
