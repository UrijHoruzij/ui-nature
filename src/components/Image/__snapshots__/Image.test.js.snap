// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component Render image 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 300px;
  height: 400px;
}

.c1 {
  width: 100%;
  height: 100%;
  border: none;
  border-radius: 4px;
  object-fit: cover;
}

<div
  className="c0"
  height="400"
  src="https://images.unsplash.com/photo-1477322524744-0eece9e79640"
  width="300"
>
  <img
    className="c1"
    src="https://images.unsplash.com/photo-1477322524744-0eece9e79640"
  />
</div>
`;
